src/app/modules/mpi/components/paciente-detalle.html

```diff
@@ -1,5 +1,6 @@
 <ng-container>
-    <plex-detail direction="{{ orientacion === 'vertical' ? 'column' : 'row' }}" size="lg" justify="{{ justificado }}">
+    <plex-detail direction="{{ orientacion === 'vertical' ? 'column' : 'row' }}" size="{{ size }}" items="{{ datos }}"
```

Comment:
```suggestion
    <plex-detail [direction]="orientacion === 'vertical' ? 'column' : 'row'" [size]="size" items="[datos]"
```

Es mala práctica usar las llaves `{{}}` para inputs de componentes. Si probas ahora el inputs items no te va a funcionar nada porque las llaves trasforman a la variable a un string.